students/kaczmarek_katarzyna/lesson_05_lists/comma_code.py

```diff
@@ -0,0 +1,17 @@
+def stringify(items):
+    if len(items) > 0:
+        return ", ".join(str(item) for item in items[:-1]) + \
```

Comment:
mozna by dodac przypadek, kiedy lista ma jedne element żebyśmy otrzymali:
`item1` zamiast `and item`